src/qt/bitcoingui.cpp

```diff
@@ -232,13 +232,14 @@ BitcoinGUI::BitcoinGUI(QWidget *parent):
     rpcConsole = new RPCConsole(this);
     connect(openRPCConsoleAction, SIGNAL(triggered()), rpcConsole, SLOT(show()));
 
-     upgrader = new UpgradeDialog(this);
-     connect(upgradeAction, SIGNAL(triggered()), upgrader, SLOT(show()));
-     connect(upgradeAction, SIGNAL(triggered()), upgrader, SLOT(upgrade()));
-     connect(downloadAction, SIGNAL(triggered()), upgrader, SLOT(show()));
-     connect(downloadAction, SIGNAL(triggered()), upgrader, SLOT(blocks()));
-
-     diagnosticsDialog = new DiagnosticsDialog(this);
+    upgrader = new UpgradeDialog(this);
+    connect(upgradeAction, SIGNAL(triggered()), upgrader, SLOT(show()));
+    connect(upgradeAction, SIGNAL(triggered()), upgrader, SLOT(upgrade()));
+    upgradeAction->setEnabled(false);
```

Comment:
I think we can hide it completely.

Reply:
yea i was thinking about it. ill see if i can hide it. cant comment out the connects or u get a entry problem with qt5core.dll as i learned ill do that next